### PR TITLE
Add EIP: Homogeneous Validator Penalty

### PR DESCRIPTION
**Pull request by skoya**
**ATTENTION: ERC-RELATED PULL REQUESTS NOW OCCUR IN [ETHEREUM/ERCS](https://github.com/ethereum/ercs)**

--

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-7570.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+eip: 7570
```
Comment (abcoathup) [inline EIPS/eip-7570.md:2]:
```suggestion
eip: 7605
```
Assigning next sequential EIP/ERC/RIP number.  7570 has already been assigned to an ERC.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Comment (skoya) [reply EIPS/eip-7570.md]: Noted, thanks.  Just pushed

**Comment by yogesh33512** [comment]
Hi Can you explain me how to get the EIP number as a initial state?

I want to submit a draft proposal but i dont have idea to how i get the eip number?

**Comment by abcoathup** [comment]
> Hi Can you explain me how to get the EIP number as a initial state?
> 
> I want to submit a draft proposal but i dont have idea to how i get the eip number?

When you submit a PR an EIP number will be issued manually by an EIP editor or associate.

File: EIPS/eip-7605.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## Security Considerations
+
+- Assessment of the risk of validators gaming the diversity metrics system.
+- Ensuring that the process of reporting hardware and software setups does not introduce new security vulnerabilities.
+
+Implementation
+
+- Development of smart contracts to handle the diversity metrics and adjusted staking rewards.
+- Guidelines for validators on how to comply with the new diversity requirements.
+- Algorithms / contracts should be released first to allow analysis of the network prior to implementation of staking penalties.
```
Comment (gaudren) [inline EIPS/eip-7605.md:61]: Instructions for the implementation should be in the Specification section.

File: EIPS/eip-7605.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+By incentivizing a diverse set of validators, Ethereum can enhance its resilience against specific risks like coordinated attacks targeting a particular type of hardware or client. This proposal balances the need for network security with the practicalities and privacy concerns of validator operators.
+
+## Backwards Compatibility
+
+This EIP should be compatible with existing Ethereum protocols. However, it will require validators to update their processes for staking and potentially their node setups.  Possible hard fork required.
+
+## Test Cases
+
+- Simulation of the revised staking rewards mechanism under various scenarios of network diversity.
+- Analysis of the effectiveness of the diversity metrics in promoting a balanced validator distribution.
```
Comment (gaudren) [inline EIPS/eip-7605.md:50]: Don't just describe what test cases you want, actually give the specific test cases.

**Review by gaudren — CHANGES_REQUESTED** [review]
EIPs are technical documents and have to contain enough information for client developers to implement. This EIP reads more like a wishlist of features, than an implementable standard.

**Pull request closed without merging**